packages/terra-markdown/src/Markdown.jsx

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import React, { PropTypes } from 'react';
 import marked from 'marked';
-import 'github-markdown-css/github-markdown.css';
+// eslint-disable-next-line import/extensions
+import 'github-markdown-css';
```

Comment:
The main attribute points to the css we want to import so we can update the syntax to this.
https://github.com/sindresorhus/github-markdown-css/blob/gh-pages/package.json#L12